style/index.html

```diff
@@ -7,7 +7,7 @@
       name="viewport"
       content="initial-scale=1,maximum-scale=1,user-scalable=no"
     />
-    <script src="https://unpkg.com/maplibre-gl@1.14.0/dist/maplibre-gl.js"></script>
+    <script src="/js/maplibre-gl.js"></script>
```

Comment:
> This branch includes a development version of maplibre-gl-js taken from maplibre/maplibre-gl-js#716, which is incompatible with the license of openstreetmap-americana.  This must be resolved prior to merging.

Instead of vendoring maplibre-gl.js, consider using NPM to install the [maplibre-gl](https://www.npmjs.com/package/maplibre-gl) package. The branch for maplibre/maplibre-gl-js#716 hasn’t been released and published yet, but you can [depend on a branch](https://docs.npmjs.com/cli/v8/configuring-npm/package-json#github-urls) as a temporary measure.

Reply:
For anyone watching this issue, help is needed with the above description.

Reply:
As of abfec69, this is resolved.